sources/Cargo.lock

```diff
@@ -1703,9 +1796,9 @@ checksum = "490cc448043f947bae3cbee9c203358d62dbee0db12107a74be5c30ccfd09771"
 
 [[package]]
 name = "memchr"
-version = "2.4.0"
+version = "2.3.4"
```

Comment:
Lots of downgrades like this. Is this a rebase gone awry, or do the new dependencies force us into older versions?

Reply:
Maybe copy from develop and overwrite the Cargo.lock, then build.